src/js/components/Accordion/index.d.ts

```diff
@@ -2,16 +2,17 @@ import * as React from 'react';
 import { AnimateType } from '../../utils';
 import { BoxProps } from '../Box';
 
-export interface AccordionProps {
+export interface AccordionProps extends BoxProps {
   activeIndex?: number | number[];
   animate?: AnimateType;
   onActive?: (activeIndexes: number[]) => void;
   multiple?: boolean;
   messages?: { tabContents?: string };
 }
 
-declare const Accordion: React.FC<BoxProps &
-  AccordionProps &
-  JSX.IntrinsicElements['div']>;
+export type AccordionExtendedProps = AccordionProps &
```

Comment:
I think we should export an interface rather than a type.

Reply:
That's a problem we discussed in the closed issue #4978 . `extends` only allow extend one item, components that extends more than that need a special form.
I suggested use a intermediate variable, something like that:

```ts
interface a extends b { }
export interface c extends a { }
```

But we agree that isn't good, so @abnersajr came with this approach using `type` and the `&`.

Reply:
There is a way to do it.
Here is an example.
https://github.com/grommet/grommet/pull/5040/files#diff-5f4f62bcd0e71612e1588e56d33792b4a74c8e011fb6b5fe9b06bb8436ddc307R23

Reply:
sure, i'll change that